test/functional/mempool_accept.py

```diff
@@ -190,7 +190,7 @@ def run_test(self):
         tx.deserialize(BytesIO(hex_str_to_bytes(raw_tx_reference)))
         # Reference tx should be valid on itself
         self.check_mempool_result(
-            result_expected=[{'txid': tx.rehash(), 'allowed': True, 'vsize': tx.get_vsize(), 'fees': { 'base': Decimal(str(0.1 - 0.05))}}],
+            result_expected=[{'txid': tx.rehash(), 'allowed': True, 'vsize': tx.get_vsize(), 'fees': { 'base': Decimal(str(0.10000000 - 0.05000000))}}],
```

Comment:
These are the values from L170 and L188. I think we should extract those to a proper variable that is also of type `Decimal`. Or at the very least this line should read `... { 'base': Decimal('0.10000000') - Decimal('0.05000000')}` IMO.

Reply:
Made the changes to remove `str()` in 7be315edd7bb5dbc4c27052b3f38d1498b51ce8b